libs/dal/src/repositories/member/member.repository.ts

```diff
@@ -136,14 +173,14 @@ export class MemberRepository extends BaseRepository<EnforceEnvironmentQuery, Me
       'invite.token': token,
     } as unknown as EnforceEnvironmentQuery;
 
-    return await this.findOne(requestQuery);
+    return await this.findOne(requestQuery as any);
```

Comment:
the type here should be `EnforceIdentifierQuery`, if TS doesn't allow, please update `EnforceEnvironmentQuery` to allow `string` as a key or this specific key `invite.token`